proto/one/historian/ingest/file_configuration.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+
+import "google/protobuf/wrappers.proto";
+
+option csharp_namespace ="ONE.Models.CSharp";
+
+message FileConfiguration
+{
+     /// Where to monitor for new files
+     string MonitorPath = 1;
+
+     // Where to monitor for new files
+     google.protobuf.StringValue MonitorFileFilter = 2;
+
+     // Where to put files that have errors
+     string ErrorPath = 3;
+
+     // Where to put files that have been processed.
+     string SuccessPath = 4;
+
+     // How long to keep files after processing
+     uint32 FileRetentionTime = 5;
```

Comment:
Something to keep in mind is that uint32 will default to zero if not provided, if zero is a valid value you will not be able to tell the difference between a field not being set and one being set to zero.  If this is important take a look at googles nullable wrappers.  We have used those elsewhere.  If it is a required property or you do not need to treat a missing value different from a default value then this is fine.

Reply:
thanks for the info.  this is a required value, so i used the protobuf type directly.   for the optional `MonitorFileFilter` i did use the wrapper.

Reply:
Sorry, I noticed that when I came back for the second review